src/impl/package/version/packageInfo.ts

```diff
@@ -29,4 +29,39 @@ export default class PackageInfo {
     );
     return packageDetails;
   }
+  public async getPackagesDetailsfromDevHub(
```

Comment:
@manisfdcsfdx  Can you add a async-retry here, these queries are notorious for slowness

Reply:
added async retry